src/main/java/ReviewResponse.java

```diff
@@ -2,6 +2,8 @@
 import lombok.Getter;
 import lombok.ToString;
 
+import java.util.List;
+
 @ToString
 @Data
```

Comment:
Violation of Single Responsibility Principle (SRP) as this class is annotated with both @ToString and @Data, indicating it is responsible for multiple functionalities. (Severity: medium)